deployment/refinery-modules/refinery/manifests/docker.pp

```diff
@@ -1,5 +1,15 @@
 class refinery::docker {
-  class { '::docker':
-    docker_users => [$::app_user]
+  if $::deployment_platform == 'aws' {
+    # Add env var pointing to remote docker host
+    file_line { "docker_host":
+      path => "/home/${app_user}/.profile",
+      line => "export DOCKER_HOST=${docker_host}",
+    }
```

Comment:
Is setting DOCKER_HOST in ~/.profile the best practice for production environments?

Reply:
The current implementation requires someone to run the `load_tools` mgmt. command to add new `ToolDefinitions` representative of `VisualizationTools` into the system. This pulls down docker images under the hood. I don't know of a better approach currently, but somehow this user needs the `DOCKER_HOST` env var set.